README.md

```diff
@@ -16,5 +16,5 @@ Solution Pack examines Advisories sent by various organisations via email with a
 
 # Next Steps
 
-| [Installation](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.1.0/docs/setup.md#installation) | [Configuration](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.1.0/docs/setup.md#configuration) | [Usage](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.1.0/docs/usage.md) | [Contents](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.1.0/docs/contents.md) |
+| [Installation](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.2.0/docs/setup.md#installation) | [Configuration](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.2.0/docs/setup.md#configuration) | [Usage](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.2.0/docs/usage.md) | [Contents](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.2.0/docs/contents.md) |
```

Comment:
please give relative path instead of FQDN

Also, please change version to 1.0.0 in readme.md

Reply:
Done